jme3-tests-template/src/com/jme3/gde/templates/tests/testsandroid/JME3TestsAndroidPanelVisual.java

```diff
@@ -110,7 +110,7 @@ private void browseButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-
         String command = evt.getActionCommand();
         if ("BROWSE".equals(command)) {
             JFileChooser chooser = new JFileChooser();
-            FileUtil.preventFileChooserSymlinkTraversal(chooser, null);
```

Comment:
Same here. Does setting the current directory to null prevent symlinks (Note: I didn not read the javadoc, I don't know what the methods do)